routes/api/user/book.js

```diff
@@ -5,83 +5,83 @@ const User = require('common/User')
 const Book = require('common/Book')()
 
 module.exports = {
-	get: function(req, res){
-		if(!req.session.passport.user){
-			return res.status(500).send()
-		}
-		let user = User(req.session.passport.user)
-		let page = Number(req.query.page)
+  get: function(req, res){
+    if(!req.session.passport.user){
+      return res.status(500).send()
+    }
+    let user = User(req.session.passport.user)
+    let page = Number(req.query.page)
 
-		user.fetchRegisteredBooks(function(err, bookList){
-			if(err){
-				return log.info(err)
-			}
+    user.fetchRegisteredBooks(function(err, bookList){
+      if(err){
+        return log.info(err)
+      }
 
-			let conditions = {
-				_id: { $in: bookList.map(function(book){ return book.bookId }) }
-			}
-			if(page){
-				// クエリパラメータにpageの指定があればそのページを返す
-				Book.fetchPage(conditions, page, function(err, books, hasNext){
-					if(err){
-						return log.info(err)
-					}
-					res.send({
-						maxCount: bookList.length,
-						books: books.map(function(book){
-							return Book.sanitizeForClient(book)
-						}),
-						hasNext: hasNext
-					})
-				})
-			}else{
-				// クエリパラメータがなければ全件を返す
-				Book.fetchAll(conditions, function(err, books){
-					if(err){
-						return log.info(err)
-					}
-					res.send({
-						maxCount: bookList.length,
-						books: books.map(function(book){
-							return Book.sanitizeForClient(book)
-						})
-					})
-				})
-			}
-		})
-	},
+      let conditions = {
+        _id: { $in: bookList.map(function(book){ return book.bookId }) }
+      }
+      if(page){
+        // クエリパラメータにpageの指定があればそのページを返す
+        Book.fetchPage(conditions, page, function(err, books, hasNext){
+          if(err){
+            return log.info(err)
+          }
+          res.send({
+            maxCount: bookList.length,
+            books: books.map(function(book){
+              return Book.sanitizeForClient(book)
+            }),
+            hasNext: hasNext
+          })
+        })
+      }else{
+        // クエリパラメータがなければ全件を返す
+        Book.fetchAll(conditions, function(err, books){
+          if(err){
+            return log.info(err)
+          }
+          res.send({
+            maxCount: bookList.length,
+            books: books.map(function(book){
+              return Book.sanitizeForClient(book)
+            })
+          })
+        })
+      }
+    })
+  },
 
-	post: function(req, res){
-		if(!req.session.passport.user){
-			return res.status(500).send()
-		}
-		let user = User(req.session.passport.user)
-		let book = req.body.newBook
+  post: function(req, res){
+    if(!req.session.passport.user){
+      return res.status(500).send()
+    }
+    let user = User(req.session.passport.user)
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Missing 'new' prefix when invoking a constructor.

*  Missing semicolon.